### ISSUE TITLE
global name 'HeatmapLayer' is not defined

### ISSUE THREAD

**Issue by carolinux** [pbugnion/gmaps#88]
I get this when I try to do
```
import gmaps
gmaps.heatmap(some data)
```

I just got the most recent version via pip

**Comment by carolinux** [comment]
added a fix for it

**Comment by pbugnion** [comment]
Ah thanks! That method is deprecated though, and probably shouldn't be used. The preferred way is 

```python
m = gmaps.Map()
heatmap_layer = gmaps.Heatmap(data=data)
m.add_layer(heatmap_layer)
m
```

**Comment by pbugnion** [comment]
Now closed thanks to PR #89 . Thanks!
